components/dialogs/AuthDialogContent/Register.vue

```diff
@@ -185,13 +192,50 @@ export default {
     avatarSrc() {
       return this.avatarData || this.prefilledData.avatarURL;
     },
-    isFormValid() {
-      const isIdValid = new RegExp(LIKECOIN_ID_REGEX).test(this.likeCoinId);
-      const isEmailValid = (
+    isIdValid() {
+      return new RegExp(LIKECOIN_ID_REGEX).test(this.likeCoinId);
```

Comment:
should reuse a same regex instance instead of new-ing `new RegExp(LIKECOIN_ID_REGEX)` everytime